Sources/WebDriver/Element.swift

```diff
@@ -107,13 +107,6 @@ public struct Element {
             session: session.id, element: id, attribute: name)).value
     }
 
-    /// Sends key presses to this element.
```

Comment:
You'll need to update:
https://github.com/thebrowsercompany/swift-webdriver/blob/4d658fdf701c190c4a8dc0957d3c8538d305fb70/Docs/SupportedAPIs.md?plain=1#L39

Reply:
Why? It's still supported, just not that overload